src/js/vic-v2/components/CropperController.jsx

```diff
@@ -131,13 +137,19 @@ export default class CropperControls extends React.Component {
     };
   }
 
-  componentWillUpdate(nextProps) {
-    if (nextProps.windowWidth !== this.props.windowWidth) {
-      const cropper = this.refs.cropper;
-      if (cropper) {
-        this.setCropBox();
```

Comment:
this is triggered as part of remounting the component so it's not longer necessary to trigger it here.